app/assets/javascripts/admin/addon/components/webhook-events.js

```diff
@@ -37,6 +41,21 @@ export default class WebhookEvents extends Component {
     } finally {
       this.loading = false;
     }
+    this.eventIds = this.filterFailedEventIds();
+    this.failedEventCount = this.eventIds.length;
+    if (this.failedEventCount === 0) {
+      this.redeliverEnabled = false;
+    }
+  }
+
+  filterFailedEventIds() {
```

Comment:
I actually think instead of these two functions it would be easier to combine them. Also if you make it a getter you can skip the `()` when using it:

```js
get failedEventIds() {
    return this.events.content
      .filter((event) => ((event.status < 200 || event.status > 299) && event.status !== 0))
      .map((event) => event.id);
  }
 ```
 
 And then it's usage becomes
 ```js
    if (!this.failedEventIds.length) {
      this.redeliverEnabled = false;
    }
```
instead of
```js
    this.eventIds = this.filterFailedEventIds();
    this.failedEventCount = this.eventIds.length;
    if (this.failedEventCount === 0) {
      this.redeliverEnabled = false;
    }
```